handbook/handbook.sop.sop.md

```diff
@@ -0,0 +1,50 @@
+---
+id: c9UtqSWj7tBL18hgOkjG5
+title: Sop
+desc: ''
+updated: 1624217315854
+created: 1624213241116
+---
+
+## Summary
+
+This is the meta SOP (Standard Operating Procedures) for SOP.
+
+## Process
+
+### Creating a new SOP
+When a SOP is first created, make sure to send a pull request to the right audience (eg. usually either the team or the community at large). If a SOP is sent to the team, everyone should be listed as a reviewer. The SOP needs to have everyone approving it before it can be merged
+
+Do not just blindly accept the SOP. When you approve the SOP, it means that you have read it, understand the action items, and agree to implement them moving forward.
+
+### Updating an existing SOP
```

Comment:
What do you think about an "executing  an SOP" section? I think this could be a valuable body of knowledge over time where you see different people executing the SOP tracked in the daily journal (and backlinked from the SOP). 

I think it would only apply to more *involved* SOPs like the release.